creation/web_base/glidein_startup.sh

```diff
@@ -1,145 +1,69 @@
-#!/bin/bash
+#!/bin/bash -xv
 
 # SPDX-FileCopyrightText: 2009 Fermi Research Alliance, LLC
 # SPDX-License-Identifier: Apache-2.0
 
-#
-# Project:
-#   glideinWMS
-#
-# File Version:
-#
+#*******************************************************************#
+#  glidein_startup.sh                                               #
+# Main Glidein script. Load all components from separate scripts,   #
+# start the Glidein up, invoke HTCondor startup, and cleanup        #
+# at the end                                                        #
+#*******************************************************************#
 
-# default IFS, to protect against unusual environment, better than "unset IFS" because works with restoring old one
+################################
+# Default IFS, to protect against unusual environment
+# better than "unset IFS" because works with restoring old one
 IFS=$' \t\n'
 
-global_args="$*"
+GLOBAL_ARGS="$*"
 # GWMS_STARTUP_SCRIPT=$0
 GWMS_STARTUP_SCRIPT="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)/$(basename "${BASH_SOURCE[0]}")"
 GWMS_PATH=""
+
+################################
 # Relative to the work directory (GWMS_DIR, gwms_lib_dir, gwms_bin_dir and gwms_exec_dir will be the absolute paths)
 # bin (utilities), lib (libraries), exec (aux scripts to be executed/sourced, e.g. pre-job)
 GWMS_SUBDIR=".gwms.d"
 
-export LANG=C
-
+################################
 # General options
-GWMS_MULTIUSER_GLIDEIN=
 # Set GWMS_MULTIUSER_GLIDEIN if the Glidein may spawn processes (for jobs) as a different user.
 # This will prepare the glidein, e.g. setting to 777 the permission of TEMP directories
 # This should never happen only when using GlExec. Not in Singularity, not w/o sudo mechanisms.
 # Comment the following line if GlExec or similar will not be used
+GWMS_MULTIUSER_GLIDEIN=
 #GWMS_MULTIUSER_GLIDEIN=true
+
 # Default GWMS log server
 GWMS_LOGSERVER_ADDRESS='https://fermicloud152.fnal.gov/log'
-
-##############################
-# Utility functions to allow the script to source functions and retrieve data stored as tarball at the end of the script itself
-
-get_data() {
-    # Retrieve the specified data, which is appended as tarball
```

Comment:
I would keep up here these utilities. These have to be available and used before using anything that was separated on a different file.